README.md

```diff
@@ -173,6 +176,13 @@ Based on this observation, reducing the domain name and doing an indexOf
 search of an (even much longer) blacklist is *much* faster than adding lists
 of .\*domain.com entries to the \*\_regex files.
 
+### rcpt accept mode
+
+By default this plugin only rejects recipients on the blacklists, and ignores
+those on the whitelists. Setting `rcpt.accept=true` makes it behave like a true
```

Comment:
I like everything except the word "true."

Reply:
easy enough ;) "true" is now gone.